hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/tasks/TestNSSummaryTaskWithOBS.java

```diff
@@ -0,0 +1,563 @@
+      keyEvent1 = new OMDBUpdateEvent.
+          OMUpdateEventBuilder<String, OmKeyInfo>()
+          .setKey(omPutKey)
+          .setValue(omPutKeyInfo)
+          .setTable(omMetadataManager.getKeyTable(getBucketLayout())
+              .getName())
+          .setAction(OMDBUpdateEvent.OMDBUpdateAction.PUT)
+          .build();
+      // PUT Key7 in Bucket1.
+      omPutKey =
+          OM_KEY_PREFIX + VOL
+              + OM_KEY_PREFIX + BUCKET_ONE +
+              OM_KEY_PREFIX + KEY_SEVEN;
+      omPutKeyInfo = buildOmKeyInfo(VOL, BUCKET_ONE, KEY_SEVEN,
+          KEY_SEVEN, KEY_SEVEN_OBJECT_ID, BUCKET_ONE_OBJECT_ID, KEY_SEVEN_SIZE);
+      keyEvent2 = new OMDBUpdateEvent.
+          OMUpdateEventBuilder<String, OmKeyInfo>()
+          .setKey(omPutKey)
+          .setValue(omPutKeyInfo)
+          .setTable(omMetadataManager.getKeyTable(getBucketLayout())
+              .getName())
+          .setAction(OMDBUpdateEvent.OMDBUpdateAction.PUT)
+          .build();
+
+      // Test DELETE Event.
+      // Delete Key1 in Bucket1.
+      String omDeleteKey =
+          OM_KEY_PREFIX + VOL
+              + OM_KEY_PREFIX + BUCKET_ONE +
+              OM_KEY_PREFIX + KEY_ONE;
+      OmKeyInfo omDeleteKeyInfo = buildOmKeyInfo(VOL, BUCKET_ONE, KEY_ONE,
+          KEY_ONE, KEY_ONE_OBJECT_ID, BUCKET_ONE_OBJECT_ID, KEY_ONE_SIZE);
+      keyEvent3 = new OMDBUpdateEvent.
+          OMUpdateEventBuilder<String, OmKeyInfo>()
+          .setKey(omDeleteKey)
+          .setTable(omMetadataManager.getKeyTable(getBucketLayout())
+              .getName())
+          .setValue(omDeleteKeyInfo)
+          .setAction(OMDBUpdateEvent.OMDBUpdateAction.DELETE)
+          .build();
+
+      // Test UPDATE Event.
+      // Resize Key2 in Bucket1.
+      String omResizeKey =
+          OM_KEY_PREFIX + VOL
+              + OM_KEY_PREFIX + BUCKET_ONE +
+              OM_KEY_PREFIX + KEY_TWO;
+      OmKeyInfo oldOmResizeKeyInfo =
+          buildOmKeyInfo(VOL, BUCKET_ONE, KEY_TWO, KEY_TWO, KEY_TWO_OBJECT_ID,
+              BUCKET_ONE_OBJECT_ID, KEY_TWO_OLD_SIZE);
+      OmKeyInfo newOmResizeKeyInfo =
+          buildOmKeyInfo(VOL, BUCKET_ONE, KEY_TWO, KEY_TWO, KEY_TWO_OBJECT_ID,
+              BUCKET_ONE_OBJECT_ID, KEY_TWO_OLD_SIZE + 100);
+      keyEvent4 = new OMDBUpdateEvent.
+          OMUpdateEventBuilder<String, OmKeyInfo>()
+          .setKey(omResizeKey)
+          .setOldValue(oldOmResizeKeyInfo)
+          .setValue(newOmResizeKeyInfo)
+          .setTable(omMetadataManager.getKeyTable(getBucketLayout())
+              .getName())
+          .setAction(OMDBUpdateEvent.OMDBUpdateAction.UPDATE)
+          .build();
+
+      OMUpdateEventBatch omUpdateEventBatch =
+          new OMUpdateEventBatch(new ArrayList<OMDBUpdateEvent>() {
+            {
+              add(keyEvent1);
+              add(keyEvent2);
+              add(keyEvent3);
+              add(keyEvent4);
+            }
+          });
```

Comment:
Use `Arrays.asList()`, similar to:

https://github.com/apache/ozone/blob/fb2caac8ba7bf646d246cc2bd3138bdf37167e66/hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/tasks/TestNSSummaryTaskWithFSO.java#L393-L396